ksql-engine/src/test/java/io/confluent/ksql/function/udf/TestUdf.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf;
+
+@UdfDescription(name="test_udf", description = "test")
+public class TestUdf {
+
+  @Udf(description = "returns the method name")
+  public String doStuffIntString(final int arg1, final String arg2) {
+    return "doStuffIntString";
+  }
+
+  @Udf(description = "returns the method name")
+  public String doStuffLongString(final long arg1, final String arg2) {
+    return "doStuffLongString";
+  }
+
+  @Udf(description = "returns the method name")
+  public String doStuffLongLongString(final long arg1, final long arg2, final String arg3) {
```

Comment:
Disable unused warnings on this class. With a comment that its invoked via reflection.